scipy/stats/_hypotests.py

```diff
@@ -666,6 +666,22 @@ def somersd(x, y=None):
     return SomersDResult(d, p, table)
 
 
+def _all_partitions(nx, ny):
+    """
```

Comment:
asking to confirm my understanding, `nx` and `ny` both need to be int, correct? I'm assuming b/c method is private we do not need to test/check is int.

Reply:
That's right.